src/test/java/com/ecore/roles/api/RolesApiTest.java

```diff
@@ -108,10 +98,10 @@ void shouldGetAllRoles() {
         RoleDto[] roles = getRoles()
                 .extract().as(RoleDto[].class);
 
-        assertThat(roles.length).isGreaterThanOrEqualTo(3);
-        assertThat(roles).contains(RoleDto.fromModel(DEVELOPER_ROLE()));
-        assertThat(roles).contains(RoleDto.fromModel(PRODUCT_OWNER_ROLE()));
-        assertThat(roles).contains(RoleDto.fromModel(TESTER_ROLE()));
+        assertThat(roles).hasSizeGreaterThanOrEqualTo(3)
```

Comment:
This code can be smaller with hasSizeGreaterThanOrEqualTo assertion method and chained method calls.